cert-manager/cert-manager-webhook-role-binding/base/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Since this resource is not a part of `cert-manager-webhook` application, this does not get tagged with its label. Should we add and overlay with another set of labels to track this through kfctl?